introduction.md

```diff
@@ -2,3 +2,8 @@ I'm RiceBen
 Lesson 1 of Mastering of Github
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?